lib/openai_ex/chat_completion.ex

```diff
@@ -9,8 +9,10 @@ defmodule OpenaiEx.ChatCompletion do
   - `:messages`
   - `:model`
   - `:frequency_penalty`
-  - `:function_call`
-  - `:functions`
+  - `:function_call` [Deprecated]
```

Comment:
I think it's ok to just delete the 'functions' and 'function_call' parameters, instead of deprecating them. They've only been in the library for a couple of weeks and not many people are using them.

Reply:
@restlessronin Done